backend/src/main/java/net/vidux/camhub/discovery/RawCameraDataFactory.java

```diff
@@ -0,0 +1,28 @@
+package net.vidux.camhub.discovery;
+
+class RawCameraDataFactory {
+  private static final String regex = "\\t";
```

Comment:
Java constant naming convention-t tartsuk be!

Reply:
Ehhez még hozzátenném, hogy érdemes ezt is érthetőbben elnevezni.
Pl.: `RAW_CAMERA_LINE_SEPARATOR`

Egyébként amíg nem reguláris kifejezésként használod, addig ez nem reguláris kifejezés (regex), hanem egy raw camera line elválasztót jelölő konstans.